test/e2e/e2e_test.go

```diff
@@ -0,0 +1,147 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/xmudrii/etcdproxy-controller/pkg/apis/etcd/v1alpha1"
+	clientset "github.com/xmudrii/etcdproxy-controller/pkg/client/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// kubeconfigPath is path to the kubernetes configuration file.
+var kubeconfigPath = filepath.Join(os.Getenv("HOME"), "/.kube/config")
+
+// newKubeClient create new Kubernetes client instance based on kubeconfig file.
+func newKubeClient(kubeconfigPath string) (*kubernetes.Clientset, error) {
```

Comment:
Is it okay to leave those functions in this file, or should I create a new file such as `util.go`?

Reply:
> Is it okay to leave those functions in this file, or should I create a new file such as util.go?

don't create a "util" anything until absolutely necessary.  Let's say 1000 lines.  They generally become a bucket of stuff.